sound/soc/sof/trace.c

```diff
@@ -44,8 +44,9 @@ static int trace_filter_parse_entry(struct snd_sof_dev *sdev, const char *line,
 	int ret;
 
 	/* ignore empty content */
-	if (!line[0])
-		return 0;
+	sscanf(line, " %n", &read);
```

Comment:
why " %n" is used?
```suggestion
	sscanf(line, "%n", &read);
```

Reply:
Space in sscanf() formatter is responsible for whitespaces consumption,
%n writes number of characters (whitespaces in this case) read by sscanf().
When input string consist only of whitespaces (read == len), then just
skip this content without error notification.